bugsnag-android-unity/gradle.properties

```diff
@@ -0,0 +1,12 @@
+GROUP=com.bugsnag
+
+POM_NAME=Bugsnag Android Unity
+POM_DESCRIPTION=Official Unity extension for Bugsnag on Android
+POM_URL=https://bugsnag.com
+POM_ARTIFACT_ID=bugsnag-android-unity
+POM_PACKAGING=aar
+
+ANDROID_BUILD_TARGET_SDK_VERSION=21
+ANDROID_BUILD_TOOLS_VERSION=25.0.2
+ANDROID_BUILD_SDK_VERSION=25
+ANDROID_BUILD_MIN_SDK_VERSION=4
```

Comment:
Did we want to increase this to 14, or leave that until we've updated to Bugsnag Android 4.0.0?

Reply:
I made a note to move the version when we upgrade to 4.0.0